assets/js/modules/ads/components/settings/SettingsEdit.stories.js

```diff
@@ -172,3 +181,74 @@ IcePaxEnabled.decorators = [
 		);
 	},
 ];
+
+export const FirstPartyModeEnabled = Template.bind( null );
+FirstPartyModeEnabled.storyName = 'FirstPartyModeEnabled';
+FirstPartyModeEnabled.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			const fpmServerRequirementsEndpoint = new RegExp(
+				'^/google-site-kit/v1/core/site/data/fpm-server-requirement-status'
+			);
+
+			fetchMock.getOnce( fpmServerRequirementsEndpoint, {
+				body: {
+					isEnabled: true,
+					isFPMHealthy: true,
+					isScriptAccessEnabled: true,
+				},
+			} );
+
+			registry.dispatch( CORE_SITE ).receiveGetFirstPartyModeSettings( {
+				isEnabled: true,
+				isFPMHealthy: true,
+				isScriptAccessEnabled: true,
+			} );
```

Comment:
We could apply DRY approach here.
```suggestion
			const fpmSettings = {
				isEnabled: true,
				isFPMHealthy: true,
				isScriptAccessEnabled: true,
			};

			fetchMock.getOnce( fpmServerRequirementsEndpoint, {
				body: fpmSettings,
			} );

			registry
				.dispatch( CORE_SITE )
				.receiveGetFirstPartyModeSettings( fpmSettings );
```